airflow/www/utils.py

```diff
@@ -99,7 +103,42 @@ def wrapper(*args, **kwargs):
         session.commit()
 
         return f(*args, **kwargs)
-    
+
+    return wrapper
+
+
+def notify_owner(f):
+    '''
+    Decorator to notify owner of actions taken on their DAGs by others
+    '''
+    @functools.wraps(f)
+    def wrapper(*args, **kwargs):
+        if request.args.get('confirmed') == "true":
+            dag_id = request.args.get('dag_id')
+            task_id = request.args.get('task_id')
+            dag = dagbag.get_dag(dag_id)
+            task = dag.get_task(task_id)
+
+            if current_user and hasattr(current_user, 'username'):
+                user = current_user.username
+            else:
+                user = 'anonymous'
+
+            if task.owner != user:
+                subject = (
+                    'Actions taken on DAG {0} by {1}'.format(
+                        dag_id, user))
+                html = 'action: <i>{0}</i><br><br>'.format(f.__name__)
+                html += '<b>Parameters</b>:<br><table>'
+                for k, v in request.args.items():
+                    if k != 'origin':
+                        html += '<tr><td>{0}</td><td>{1}</td></tr>'.format(k, v)
```

Comment:
I'm not big in seeing html in python code. Alternatives are `dataframe.to_html(dict())` or using jinja templating